ui/app/models/pki/certificate/base.js

```diff
@@ -49,7 +49,8 @@ export default class PkiCertificateBaseModel extends Model {
   // Parsed from cert in serializer
   @attr('number', { formatDate: true }) notValidAfter;
   @attr('number', { formatDate: true }) notValidBefore;
-  @attr('string') uriSans;
+  @attr('string', { label: 'Subject Alternative Names' }) uriSans;
```

Comment:
If you were to choose one, I'd maybe do dnsSans as the SANs, and call uri SANs, URI SANs.

Reply:
So label this `URI SANS` and then update `altNames` below (which is dnsNames to `Subject Alternative Names` ?

Reply:
Yeah, I think so :-)